### PR TITLE
Adding mail username as apikey

### PR DESCRIPTION
**Pull request by rjshrjndrn**
We're using azure sendgrid apis and the usename is same
"apikey" for ever. So moving it to all.yaml. Anybody can override this with own variable.

### CODE REVIEW

File: ansible/inventory/env/group_vars/all.yml [hunk 1]
```diff
@@ -660,3 +660,6 @@ dp_postgres_username: analytics
 
 # Will enable cassandra cluster if number of cassandra nodes > 1
 cassandra_cluster_size: "{{ groups['cassandra'] | length }}"
+
+# Azure sendgrid mail server apitoken username
+mail_server_username: "apikey"
```
Comment (keshavprasadms) [inline ansible/inventory/env/group_vars/all.yml:665]: Why are we defining a default here?

Comment (keshavprasadms) [reply ansible/inventory/env/group_vars/all.yml:665]: If its only the username to avoid template errors, we should have password and host fields also.

Comment (rjshrjndrn) [reply ansible/inventory/env/group_vars/all.yml:665]: this is a constant for sendgrid. That'll be for the default installations. If you want to override you can do it in common.yaml

Comment (keshavprasadms) [reply ansible/inventory/env/group_vars/all.yml:665]: Okay, please update the comment for this as default value used in case of sendgrid

Comment (rjshrjndrn) [reply ansible/inventory/env/group_vars/all.yml:665]: sure

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**